src/miner.cpp

```diff
@@ -144,7 +143,7 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
     int nPackagesSelected = 0;
     int nDescendantsUpdated = 0;
 
-    addPackageTxs<VeriBlock::poptx_priority<ancestor_score>>(nPackagesSelected, nDescendantsUpdated, *pindexPrev);
+    addPackageTxs<ancestor_score>(nPackagesSelected, nDescendantsUpdated, *pindexPrev);
```

Comment:
you can remove last argument (prev index)

Reply:
fixed